docs/cli/overview.md

```diff
@@ -0,0 +1,95 @@
+ms.date: 06/26/2025
+ms.topic: overview
+ms.custom:
+  - ai-gen-docs-bap
+  - ai-gen-title
+  - ai-seo-date:06/26/2025
+  - ai-gen-description
+---
+
+# Aspire CLI Overview
+
+The Aspire CLI (`aspire` command) is a cross-platform tool that provides command-line functionality to create, manage, run, and publish polyglot Aspire projects. Use the Aspire CLI to streamline development workflows and coordinate services for distributed applications.
+
+The Aspire CLI is an interactive-first experience.
+
+- [Install .NET Aspire CLI.](install.md)
+- [`aspire` command reference.](../cli-reference/aspire.md)
+
+## Use templates
+
+[_Command reference: `aspire new`_](../cli-reference/aspire-new.md)
+
+The `aspire new` command is an interactive-first CLI experience, and is used to create one or more Aspire projects. As part of creating a project, Azure CLI ensures that the latest Aspire project templates are installed into the `dotnet` system.
+
+<!-- Add asciinema here -->
+
+Use the `aspire new` command to create an Aspire project from a list of templates. Once a template is selected, the name of the project is set, and the output folder is chosen, `aspire` downloads the latest templates and generates one or more projects.
+
+While command line parameters can be used to automate the creation of an Aspire project, the Aspire CLI is an interactive-first experience.
+
+## Start the app host
+
+[_Command reference: `aspire run`_](../cli-reference/aspire-run.md)
+
+The `aspire run` command runs the AppHost project in development mode, which configures the Aspire environment, builds and starts resources defined by the app host, launches the web dashboard, and prints a list of endpoints.
+
+When `aspire run` starts, it searches the current directory for an AppHost project. If a project isn't found, the sub directories are searched until one is found. If no AppHost project is found, Aspire stops. Once a project is found, Aspire CLI takes the following steps:
+
+- Installs or verifies that Aspire's local hosting certificates are installed and trusted.
+- Builds the AppHost project.
+- Starts the app host and its resources.
+- Starts the dashboard.
+
+The following snippet is an example of the output displayed by the `aspire run` command:
+
+```Aspire CLI
+Dashboard:  https://localhost:17178/login?t=17f974bf68e390b0d4548af8d7e38b65                                         
+                                                                                                                    
+    Logs:  /home/vscode/.aspire/cli/logs/apphost-1295-2025-07-14-18-16-13.log                                       
+                                                              
+Endpoints:  webfrontend has endpoint https://localhost:7294   
+            webfrontend has endpoint http://localhost:5131   
+            apiservice has endpoint https://localhost:7531   
+            apiservice has endpoint http://localhost:5573   
```

Comment:
This list of endpoints will not be shown if you aren't running in a Devcontainer/Codespace (may want to mention that).